plasmapy/formulary/tests/test_magnetostatics.py

```diff
@@ -39,7 +39,7 @@ def test_value2(self):
     def test_repr(self):
```

Comment:
```suggestion
    @pytest.mark.skipif(astropy.__version__ < "5.3.0", "change in unit representation")
    def test_repr(self):
```
What would you think of setting this test to run only for more recent versions of Astropy?  We'd need an `import astropy` at the top of the file too.

Reply:
Good thinking. Should we plan on bumping the version in the near future?

Reply:
We generally try to support versions of Astropy that have been released within the past ∼2 years, so it'll be a while before we require Astropy ≥ 5.3.0 since it hasn't been released yet.